packages/manager/modules/anthos/src/translations/Messages_fr_FR.json

```diff
@@ -1 +1,13 @@
-{}
+{
+  "anthos_tenants": "Anthos",
+  "anthos_hpc_title": "Hosted Private Cloud",
+  "anthos_tenants_title": "Anthos Platform",
+  "anthos_tenants_guides": "Guides",
+  "anthos_tenants_guides_all": "Tous les guides",
+  "anthos_tenants_grid_create_tenant": "Commander un hôte",
+  "anthos_tenants_grid_headers_serviceName": "serviceName",
+  "anthos_tenants_grid_headers_displayName": "displayName",
+  "anthos_tenants_grid_headers_location": "location",
+  "anthos_tenants_grid_headers_version": "version",
+  "anthos_tenants_grid_headers_vrackId": "vrackId"
```

Comment:
Is this content legit ? We will display those properties keys like that ?

Reply:
Yep, we don't have translation for them.

Reply:
I sent a request to know if it must stay like that.

Reply:
Fixed